var/spack/repos/builtin/packages/minimd/package.py

```diff
@@ -0,0 +1,73 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+import glob
+import tarfile
+
+from spack import *
+
+
+class Minimd(MakefilePackage):
+    """Proxy Application. A simple proxy for the force computations
+       in a typical molecular dynamics applications.
+    """
+
+    homepage = "http://mantevo.org"
+    url      = "http://mantevo.org/downloads/releaseTarballs/miniapps/MiniMD/miniMD_1.2.tgz"
+
+    version('1.2', '893ef1ca5062e32b43a8d11bcfe1a056')
+
+    depends_on('mpi')
+
+    @property
+    def build_targets(self):
+        targets = [
+            'LINK={0}'.format(self.spec['mpi'].mpicxx),
+            'CC={0}'.format(self.spec['mpi'].mpicxx),
+            'CCFLAGS={0} -DMPICH_IGNORE_CXX_SEEK -DNOCHUNK'.format(
+                self.compiler.openmp_flag),
+            '--directory=miniMD_ref',
```

Comment:
This just means that the Makefile resides in `miniMD_ref`? It might be better to use:
```python
build_directory = 'miniMD_ref'
```
instead. This tells Spack to change to this directory before running `make`.